### PR TITLE
Added Terraform for cluster and load scripts

### PR DESCRIPTION
**Pull request by jonnyshaw89**
This contains the terraform to create an Elasticsearch cluster and some scripts to load data into the cluster

### CODE REVIEW

File: elasticsearch.tf [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  access_policies = <<CONFIG
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": "es:*",
+            "Principal": "*",
+            "Effect": "Allow",
+            "Resource": "arn:aws:es:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:domain/jonshaw-suggest/*",
```
Comment (kierenbeckett) [inline elasticsearch.tf]: Should this have jonshaw hardcoded in it?

File: data_sets.tf [hunk 2]
```diff
@@ -0,0 +1,6 @@
+resource "null_resource" "countries" {
+  provisioner "local-exec" {
+    working_dir = "${path.module}/scripts/"
+    command     = "load-countries.sh https://${aws_elasticsearch_domain.suggest.endpoint}"
+  }
+}
```
Comment (kierenbeckett) [inline data_sets.tf]: Do we need the other 2 scripts here?

File: data/countries-bulk.json [hunk 3]
```diff
@@ -0,0 +1,398 @@
+{"index" : {}}
```
Comment (kierenbeckett) [inline data/countries-bulk.json:1]: Should we commit these, rather than just generating them temporarily when creating the index?

**Review by kierenbeckett — CHANGES_REQUESTED** [review]
Given this is a static data set, we should probably tweak the shard settings.

File: data/countries-bulk.json (hunk 3 above)
Comment (jonnyshaw89) [reply data/countries-bulk.json]: Yeh, still in progress really

Comment (jonnyshaw89) [reply data/countries-bulk.json:1]: I was going to generate them on the fly, but they take ages!

**Review by kierenbeckett — APPROVED** [review]

**Review by dwyeradam — APPROVED** [review]

**Pull request merged**